vm/os-windows.cpp

```diff
@@ -304,6 +304,54 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+static volatile bool stop_on_ctrl_break = false;
+static HANDLE ctrl_break_thread = NULL;
+
```

Comment:
I totally support the sentiment, but would it be right to have a separate Ctrl-Break handler for every VM? Currently it **is** a global application-level service, even if its current implementation only interrupts the first VM in the global list of VMs.